app/routes/agenda-items/agenda-item/session.ts

```diff
@@ -0,0 +1,11 @@
+import Route from '@ember/routing/route';
+import AgendaItemModel from 'frontend-burgernabije-besluitendatabank/models/agenda-item';
+
+export default class AgendaItemsAgendaItemSessionRoute extends Route {
+  async model() {
+    return this.modelFor('agenda-items.agenda-item') as {
+      agendaItem: AgendaItemModel;
+      agendaItemOnSameSession: AgendaItemModel[];
```

Comment:
**Blocker:** You cannot use  `agendaItemOnSameSession` from `agenda-items.agenda-item` model, because it only contains maximum 4 entries. 

Proposals: 
Move filter and slice in from agenda-item route to controller and returns the full list of agenda items as `  agendaItemOnSameSession` 
```
agendaItemOnSameSessionRaw
      .filter((item) => item.id !== agendaItem.id)
      .sort(sortObjectsByTitle)
      .slice(0, 4);
```